README.md

```diff
@@ -42,5 +42,32 @@ want to start with
 [newbie-friendly open tickets](https://issues.jenkins-ci.org/issues/?jql=project%20%3D%20JENKINS%20AND%20status%20in%20%28Open%2C%20Reopened%29%20AND%20component%20%3D%20filesystem_scm-plugin%20AND%20labels%20%3D%20newbie-friendly)
 first (if there are any).
 
+# License
+This Jenkins plugin is licensed under the
```

Comment:
No need to print a full license here. Just use `[MIT License](./LICENSE.txt)` to link the file